artemis-chart/templates/monitor-deployment.yaml

```diff
@@ -0,0 +1,173 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: monitor
+    app: {{ .Release.Name }}
+  name: monitor
+spec:
+  replicas: {{ .Values.replicaCount | default 1 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      creationTimestamp: null
+      labels:
+        io.kompose.service: monitor
+      annotations:
+        checksum/config: {{ include (print $.Template.BasePath "/configmap.yaml") . | sha256sum }}
+    spec:
+      initContainers:
+      - name: populate-pvc
+        image: busybox
+        volumeMounts:
+        - mountPath: /configmaps
+          name: monitor-configmap
+        - mountPath: /pvc
+          name: monitor-pvc
+          subPath: configs
+        command: ['sh', '-c', 'cp -u /configmaps/logging.yaml /configmaps/mon-services.conf /pvc/']
+      - name: wait-for-service
+        image: busybox
+        command: ['sh', '-c', 'until nc -z {{ .Values.rabbitmqHost }} {{ .Values.rabbitmqPort}}; do echo waiting for services; sleep 10; done;']
```

Comment:
missing a wait on backend and frontend for monitor

Reply:
yes, and we also need a wait on backend for redis, if the comment on the monitor and backend entrypoints is applied.

Reply:
as discussed, redis block not needed. Code is fine as-is.